Sources/JavaScriptKit/FundamentalObjects/JSFunction.swift

```diff
@@ -178,123 +115,3 @@ fileprivate func invokeJSFunction(_ jsFunc: JSFunction, arguments: [ConvertibleT
     }
     return result
 }
-
-/// `JSClosure` represents a JavaScript function the body of which is written in Swift.
-/// This type can be passed as a callback handler to JavaScript functions.
-/// Note that the lifetime of `JSClosure` should be managed by users manually
-/// due to GC boundary between Swift and JavaScript.
-/// For further discussion, see also [swiftwasm/JavaScriptKit #33](https://github.com/swiftwasm/JavaScriptKit/pull/33)
-///
-/// e.g.
-/// ```swift
-/// let eventListenter = JSClosure { _ in
-///     ...
-///     return JSValue.undefined
-/// }
-///
-/// button.addEventListener!("click", JSValue.function(eventListenter))
-/// ...
-/// button.removeEventListener!("click", JSValue.function(eventListenter))
-/// eventListenter.release()
-/// ```
-///
-public class JSClosure: JSFunction {
-    static var sharedFunctions: [JavaScriptHostFuncRef: ([JSValue]) -> JSValue] = [:]
-
-    private var hostFuncRef: JavaScriptHostFuncRef = 0
-
-    private var isReleased = false
-    
-    /// Instantiate a new `JSClosure` with given function body.
-    /// - Parameter body: The body of this function.
-    public init(_ body: @escaping ([JSValue]) -> JSValue) {
-        // 1. Fill `id` as zero at first to access `self` to get `ObjectIdentifier`.
-        super.init(id: 0)
-        let objectId = ObjectIdentifier(self)
-        let funcRef = JavaScriptHostFuncRef(bitPattern: Int32(objectId.hashValue))
-        // 2. Retain the given body in static storage by `funcRef`.
-        Self.sharedFunctions[funcRef] = body
-        // 3. Create a new JavaScript function which calls the given Swift function.
-        var objectRef: JavaScriptObjectRef = 0
-        _create_function(funcRef, &objectRef)
-
-        hostFuncRef = funcRef
-        id = objectRef
-    }
-    
-    /// A convenience initializer which assumes that the given body function returns `JSValue.undefined`
-    convenience public init(_ body: @escaping ([JSValue]) -> ()) {
```

Comment:
Can we keep this overload for at least one minor version as deprecated? This would give people some time to migrate away from it.